.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +21,7 @@
 ## Checklist:
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] My code follows the [code style](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#coding-conventions) of this project.
+- [ ] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
```

Comment:
I see that number of checkboxes marked as true is shown in pull request.
This looks like a *checklist* for me. Because of that I propose that we change the Checklist so that we always want to have all checkboxes checked:

```
[ ] I have reviewed (and updated if needed) the documentation regarding this change
```

Reply:
Updated